lib/ember-cli/app_win.rb

```diff
@@ -0,0 +1,31 @@
+module EmberCLI
+  class AppWin < EmberCLI::App
+
+    def ember_path
+      @ember_path ||= app_path.join("node_modules", ".bin", "ember").tap do |path|
+        fail <<-MSG.strip_heredoc unless path.exist?
+          No local ember executable found. You should run `npm install`
+          inside the #{name} app located at #{app_path}
+        MSG
+      end
+    end
+
+    def symlink_to_assets_root
+      exec ("cmd.exe /c \"mklink /J #{assets_path.join(name).to_s.gsub('/', '\\')} #{dist_path.join("assets").to_s.gsub('/', '\\')}\"")
+    rescue Errno::EEXIST
+      # Sometimes happens when starting multiple Unicorn workers.
+      # Ignoring...
+    end
+
+    def command(options={})
+      watch = options[:watch] ? "--watch" : ""
+
+      "\"#{ember_path}.cmd\" build #{watch} --environment #{environment} --output-path \"#{dist_path}\" #{log_pipe}"
```

Comment:
@swelham is right. Windows doesn't understand *hashbang* (`#!`) executables, so npm creates a `.cmd` wrapper that basically calls `node.exe <script>`. So if an npm creates a binstub on windows, then it also creates a `.cmd` file so that it can be used as an executable. Ruby does the same thing, except as batch scripts.